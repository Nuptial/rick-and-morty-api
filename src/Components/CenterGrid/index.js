import {
  Grid
} from "@mui/material";

const CenterGrid = ({ children }) => {
  return (
    <Grid
      container
      spacing={0}
      direction="column"
      alignItems="center"
      justifyContent="center"
      style={{ minHeight: "100vh" }}
    >
      {children}
    </Grid>
  );
};

export default CenterGrid;