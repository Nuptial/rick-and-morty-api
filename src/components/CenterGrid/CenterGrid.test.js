import renderer from 'react-test-renderer';

import CenterGrid from '../CenterGrid';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<CenterGrid />).toJSON();
  expect(tree).toMatchSnapshot();
});