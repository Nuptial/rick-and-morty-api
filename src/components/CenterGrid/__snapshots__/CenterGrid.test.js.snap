// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-l6hioc-MuiGrid-root"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
/>
`;
